ddtrace/tracer/spancontext.go

```diff
@@ -87,9 +87,9 @@ type SpanContext struct {
 
 	// the below group should propagate only locally
 
-	trace  *trace // reference to the trace that this span belongs too
-	span   *Span  // reference to the span that hosts this context
-	errors int32  // number of spans with errors in this trace
+	trace  *trace       // reference to the trace that this span belongs too
+	span   *Span        // reference to the span that hosts this context
+	errors atomic.Int32 // number of spans with errors in this trace
```

Comment:
```suggestion
	errors *atomic.Int32 // number of spans with errors in this trace
```

I would rather use pointer here.

Reply:
Done in 1da44f72fbf5d077010173eb81ab0fa4271d0518.